source/system/io/File.ooc

```diff
@@ -278,15 +278,27 @@ File: abstract class {
 	rename: static func (originalPath, newPath: String) {
 		rename(originalPath toCString(), newPath toCString())
 	}
-	getCwd: static func -> String {
-		ooc_get_cwd()
+	createDirectory: static func ~file (path: String) -> Int {
```

Comment:
Not really, but it is the helper function that `createDirectories` uses, and it's different for the unix and win32 implementations. We can probably make it better though, but not now.